.github/workflows/pull-request.yml

```diff
@@ -11,32 +11,14 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
         
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-    
-      - name: Cache Node dependencies
-        uses: actions/cache@v4
-        id: yarn-cache
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-      
-      - if: ${{ steps.yarn-cache.outputs.cache-hit != 'true' }}
-        name: List the state of node modules
-        continue-on-error: true
-        run: yarn list
-
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
           node-version: 20
+          cache: "yarn"
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
-
+        run: yarn workspaces focus -A
```

Comment:
Is there any advantage of using `yarn workspaces focus -A` over  `yarn install --frozen-lockfile` as we don't have multiple workspaces?

Reply:
Quoting the [documentation](https://yarnpkg.com/cli/workspaces/focus),
```
If the -A,--all flag is set, the entire project will be installed. Combine with --production to replicate the old yarn [install](https://yarnpkg.com/cli/install) --production.
```
It's the Yarn 4 equivalent of `yarn install`. But I guess I can restore it.

Reply:
Done, `yarn install --frozen-lockfile` everywhere